doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -27,6 +27,12 @@ If you do not use Newsletter functionality, then please make sure to run the fol
 bin/console pimcore:documents:cleanup newsletter
 ```
 
+If you use the Newsletter functionality please make sure to add the `pimcore/compatibility-bridge-v10` to your composer.json  
+```bash
+composer require pimcore/compatibility-bridge-v10
```

Comment:
```suggestion
Please make sure to add the `pimcore/compatibility-bridge-v10` to your composer.json  
```bash
composer require --no-update pimcore/compatibility-bridge-v10
```
I think this should be the default step on upgrade as it provides compatibility for more classes. wdyt?

Reply:
and this should move to the "PRIOR THE UPDATE" section.

Reply:
moved the section to "Preparing for v11" guide with changes suggested above https://github.com/pimcore/pimcore/pull/14902/commits/ae34126ddd2389ab9da39e9ba79f57bf2e166ed6